.github/sync_labels.py

```diff
@@ -408,7 +408,17 @@ def get_commits(self):
             return self._commits
 
         self._commits = self.view('commits')
-        self._commit_date = max( com['committedDate'] for com in self._commits )
+
+        # ignore merge commits with the develop branch for _commit_date except positive review is set
+        date_commits = list(self._commits)
+        if Status.positive_review.value not in self.get_labels():
+            for com in date_commits:
```

Comment:
```suggestion
            for com in self._commits:
```
I think removing elements from the list you're iterating over can sometimes cause weird behaviour such as skipping elements.

You could also do this more compactly with a list comprehension or `filter` but then you couldn't have the debug messages.